### ISSUE TITLE
Delete Posts by URL - Add tests

### ISSUE THREAD

**Issue by sudar** [sudar/bulk-delete#251]
## Tests
- [ ] ~Create helper function to create posts by url~ its not meaning full.
- [x] Create helper function to retrieve posts by url
- [x] Add tests to delete one valid url
- [x] Add tests to delete one invalid url
- [x] Add tests to delete one valid url and one invalid url
- [x] Add tests to delete more than one valid url
- [x] Add tests to test deleting posts permanently ( at least 2 test cases)

Once this task is done, the helper functions should be moved to the tests library repo.

While sending this task for code review create a new issue for moving the helper functions to the the test library repo

**Comment by sudar** [comment]
@rajanit2000 

Try out the solution that @mariadanieldeepak gave and let us know if you are still face any issues.

**Comment by rajanit2000** [comment]
@sudar Yes its fixed
